ReactAndroid/src/main/java/com/facebook/react/devsupport/InspectorPackagerConnection.java

```diff
@@ -269,7 +266,7 @@ public void close() {
       if (mWebSocket != null) {
         try {
           mWebSocket.close(1000, "End of session");
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
Probably shouldn't be catching all Exceptions, the people responsible for maintaining this code have wanted more precise error handling in the past